core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.html

```diff
@@ -153,6 +153,20 @@
     </span>
   </button>
 </div>
+<div *ngIf="modifyTranslationOpportunity && !opportunity" class="oppia-translate-modal-footer modal-footer">
+  <button class="btn btn-default"
+          (click)="close()">
+    Cancel
+  </button>
+  <button class="btn btn-success oppia-upload-translated-text-button e2e-test-save-button"
```

Comment:
Similarly why do we need another **Save** button? Can't we use the existing button?

Reply:
We need another save button because it serves different functionality in this use case, which is just to resolve the modal with the written translation. The CD use case is completely different.

Reply:
I see. Got it. Thanks!